arkouda/strings.py

```diff
@@ -828,6 +828,26 @@ def save(self, prefix_path : str, dataset : str='strings_array',
         self.bytes.save(prefix_path=prefix_path, 
                                     dataset='{}/values'.format(dataset), mode=mode)
 
+    def is_registered(self) -> np.bool_:
+        """
+        Return True iff the object is contained in the registry
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        bool
```

Comment:
Not sure why you are returning np.bool_ vs a Python bool?

Reply:
I chose to do it out of consistency with the other code. Like `def is_sorted(pda : pdarray) -> np.bool_:` in `pdarrayclass.py`. I wasn't exactly sure why `np.bool_` was chosen but figured I should probably follow suit. I didn't see a good reason to use it in `is_sorted` but not here, but I can certainly change it if there is

Reply:
ah gotcha, cool, makes sense, was curious, that's all, no need to change.

Reply:
That's totally reasonable! If @mhmerrill wanted to weigh in on the original design choice (and if it makes sense to use `np.bool_` here), I'd be interested to hear it